assignment.js

```diff
@@ -65,6 +69,11 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+let numbers = [3, 48, 29, 12, 18]
+const currentLargestNumber = Math.max(...numbers)
+
+console.log(currentLargestNumber)
+
 
```

Comment:
@rlou88 Good job using the es6 syntax spread operator and using the javascript math library.  Great Job take a look at this alternate approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L74)